### PR TITLE
Tracing Announcement

### PR DESCRIPTION
**Pull request by BenWilson2**
Add a blog for the tracing announcement

### CODE REVIEW

**Review by WeichenXu123 — APPROVED** [review]
LGTM!

File: website/blog/2024-06-10-mlflow-tracing/tracing-ui.gif
Comment (daniellok-db) [inline website/blog/2024-06-10-mlflow-tracing/tracing-ui.gif:1]: could we update the GIF with the latest UI fixes? it looks slightly better now!

**Review by B-Step62 — APPROVED** [review]
LGTM!

File: website/blog/2024-06-10-mlflow-tracing/index.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+```python
+import os
+from langchain.prompts import PromptTemplate
+from langchain_openai import OpenAI
+import mlflow
+
+assert "OPENAI_API_KEY" in os.environ, "Please set your OPENAI_API_KEY environment variable."
+
+mlflow.set_tracking_uri("http://localhost:5000")
```
Comment (harupy) [inline website/blog/2024-06-10-mlflow-tracing/index.md:43]: Can we remove this line or add instructions on how to run the tracking server? I'd remove it.

File: website/blog/2024-06-10-mlflow-tracing/index.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+
+> NOTE: MLflow Tracing has been released in MLflow 2.14.0 and is not available in previous versions.
+
+## Introducing MLflow Tracing
+
+Tracing is a critical aspect of understanding and optimizing complex applications, especially in the realm of machine learning and artificial intelligence.
+With the release of MLflow Tracing, you can now easily capture, visualize, and analyze detailed execution traces of your GenAI applications.
+This new feature aims to provide greater visibility and control over your applications' performance and behavior, aiding in everything from fine-tuning to debugging.
+
+## What is MLflow Tracing?
```
[the remark this answers is not in the document]
Comment (BenWilson2) [reply website/blog/2024-06-10-mlflow-tracing/index.md:21]: Excellent idea! I'll link that word too :D

File: website/blog/2024-06-10-mlflow-tracing/tracing-ui.gif
Comment (BenWilson2) [reply website/blog/2024-06-10-mlflow-tracing/tracing-ui.gif:1]: Yeah it really DOES look better, doesn't it? ;) Great work on it, by the way!! 
(updated with a new .gif)

**Review by harupy — APPROVED** [review]
LGTM!

**Pull request merged**